images/spin-multi-trigger-app/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM --platform=${BUILDPLATFORM} rust:1.73 AS build
```

Comment:
I feel strongly that we should test using `spin registry push`

Reply:
yes, that is on my todo list for next week (also discuss and cover more k8s distros probably).

Reply:
there is a ticket for it already. I will pick it up next. https://github.com/spinkube/containerd-shim-spin/issues/37